github/integration_test.go

```diff
@@ -248,6 +248,48 @@ var _ = Describe("GitHub Provider", func() {
 		getSpec := newGithubRepositorySpec(getRepo.APIObject().(*github.Repository))
 		postSpec := newGithubRepositorySpec(repo.APIObject().(*github.Repository))
 		Expect(getSpec.Equals(postSpec)).To(BeTrue())
+
+		// User repo case
+		// Generate a repository name which doesn't exist already
+		testRepoName = fmt.Sprintf("test-repo-%03d", rand.Intn(1000))
+		for findRepo(repos, testRepoName) != nil {
+			testRepoName = fmt.Sprintf("test-repo-%03d", rand.Intn(1000))
+		}
+
+		// We know that a repo with this name doesn't exist in the organization, let's verify we get an
+		// ErrNotFound
+		userLogin := os.Getenv("GITHUB_TEST_USER")
```

Comment:
Check for this env var up the top and bail if missing as is done w/ `GITHUB_TOKEN`